client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkShuffleUtils.java

```diff
@@ -123,4 +127,15 @@ public static Configuration getRemoteStorageHadoopConf(
     }
     return readerHadoopConf;
   }
+
+  public static Set<String> getDataPlacementTags(SparkConf sparkConf) {
+    Set<String> placementTags = new HashSet<>();
```

Comment:
Could we give the variable a better name?

Reply:
server assignment tag?